client/reader/start/style.scss

```diff
@@ -308,176 +308,179 @@
 	text-transform: uppercase;
 }
 
-.has-image {
-
-	.reader-start-post-preview__featured-label {
-		color: $white;
-		background: none;
-		font-size: 10px;
-		letter-spacing: .03em;
-		padding: 5px 0 0 3px;
-		position: absolute;
-		top: 0;
-		text-transform: uppercase;
-		z-index: 1;
-	}
-
-	.reader-start-post-preview__featured-image {
-		height: 80px;
-	}
-
-	.reader-start-post-preview__post-content {
-		padding: 12px 10px 20px;
-		margin-top: 0;
-	}
-
-	.reader-start-post-preview__featured-image {
-		background-size: cover;
-		border-top-left-radius: 1px;
-		border-top-right-radius: 1px;
+.reader-start-card {
```

Comment:
Isn't it better to be bare? ([sauce](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#nesting)